ext/cl_khr_command_buffer.asciidoc

```diff
@@ -1516,6 +1523,172 @@ New errors:
+
+_sync_point_wait_list_, _num_sync_points_in_wait_list_ Specify
+synchronization-points that need to complete before this
+particular command can be executed.
+
+If _sync_point_wait_list_ is `NULL`, _num_sync_points_in_wait_list_
+must be 0. If _sync_point_wait_list_ is not `NULL`, the list of
+synchronization-points pointed to by _sync_point_wait_list_ must be
+valid and _num_sync_points_in_wait_list_ must be greater than 0.
+The synchronization-points specified in _sync_point_wait_list_ are
+*device-side* synchronization-points. The command-buffer associated
+with synchronization-points in _sync_point_wait_list_ must be the same
+as _command_buffer_. The memory associated with _sync_point_wait_list_
+can be reused or freed after the function returns.
+
+_sync_point_ Returns a synchronization-point ID that identifies this particular
+command. Synchronization-point objects are unique and can be used to
+identify this command later on. _sync_point_ can be `NULL` in which case it
+will not be possible for the application to record a wait for this command to
+complete. If the _sync_point_wait_list_ and the _sync_point_ arguments are not
+`NULL`, the _sync_point_ argument should not refer to an element of the
+_sync_point_wait_list_ array.
+
+_mutable_handle_ Returns a handle to the command. Handle is unused by
+this extension and must be passed as `NULL`.
+
+{clCommandSVMMemcpyKHR} returns {CL_SUCCESS} if the function is executed
+successfully. Otherwise, it returns the errors defined by
+{clEnqueueSVMMemcpy} except:
+
+{CL_INVALID_COMMAND_QUEUE} is replaced with:
+
+* {CL_INVALID_COMMAND_QUEUE} if _command_queue_ is not `NULL`.
+
+{CL_INVALID_CONTEXT} is replaced with:
+
+* {CL_INVALID_CONTEXT} if the context associated with _command_queue_,
+  _command_buffer_, and _kernel_ are not the same.
+
+{CL_INVALID_EVENT_WAIT_LIST} is replaced with:
+
+* {CL_INVALID_SYNC_POINT_WAIT_LIST_KHR} if _sync_point_wait_list_ is `NULL` and
+  _num_sync_points_in_wait_list_ is > 0, or _sync_point_wait_list_ is not
+  `NULL` and _num_sync_points_in_wait_list_ is 0, or if
+  synchronization-point objects in _sync_point_wait_list_ are not valid
+  synchronization-points.
+
+New errors:
+
+* {CL_INVALID_COMMAND_BUFFER_KHR} if _command_buffer_ is not a valid
+  command-buffer.
+
+* {CL_INVALID_OPERATION} if _command_buffer_ has been finalized.
+
+* {CL_INVALID_VALUE} if _mutable_handle_ is not `NULL`.
+
+
+The function
+
+include::{generated}/api/protos/clCommandSVMMemFillKHR.txt[]
+
+Records a command to fill a region in SVM with a pattern of a given pattern size.
+
+_command_buffer_ Refers to a valid command-buffer object.
+
+_command_queue_ Specifies the command-queue the command will be recorded to.
+Parameter is unused by this extension as only a single command-queue is
+supported and **must** be `NULL`.
+
+_svm_ptr_ is a pointer to a memory region that will be filled with
+_pattern_. It must be aligned to _pattern_size_ bytes.
```

Comment:
Similar to my comment for the SVM memcpy, can this be any arbitrary host pointer, or must it be an SVM allocation if the device executing the command buffer does not support system SVM?